src/test/resources/data/typicalDataBundle.json

```diff
@@ -1880,7 +1880,7 @@
       "style": "DANGER",
       "targetUser": "GENERAL",
       "title": "A deprecation note",
-      "message": "Deprecation happens in three minutes",
+      "message": "<p>Deprecation happens in three minutes</p>",
```

Comment:
Just curious about what is the reason for adding the `<p>` tag in the databundle?

Reply:
Since the message input is a rich text editor, the notification message is saved as HTML instead of pure text; I've changed the data bundle to reflect this.